src/container/element-list/element-container.tsx

```diff
@@ -38,7 +39,7 @@ export class ElementContainer extends React.Component<ElementContainerProps> {
 				onChange={AlvaUtil.noop}
 				placeholderHighlighted={props.element.getPlaceholderHighlighted()}
 				state={getState(props.element, store)}
-				title={props.element.getName()}
+				title={props.element.getRole() === ElementRole.Root ? 'Page' : props.element.getName()}
```

Comment:
Let's use the page name for the root element, should be implemented in `Element.prototype.getName`